backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Mocks/Utils/SingleRightUtil.cs

```diff
@@ -0,0 +1,29 @@
+using System.Text.Json;
+using Altinn.AccessManagement.UI.Core.Models.SingleRight.CheckDelegationAccess;
+using Altinn.AccessManagement.UI.Mocks.Mocks;
+
+namespace Altinn.AccessManagement.UI.Mocks.Utils
+{
+    public class SingleRightUtil
+    {
+        public static List<DelegationAccessCheckResponse> GetMockedDelegationAccessCheckResponses(AccessLevel accessLevel)
+        {
+            string? unitTestFolder = Path.GetDirectoryName(new Uri(typeof(SingleRightClientMock).Assembly.Location).LocalPath);
+
+            string fileName = Convert.ToString(accessLevel);
+
+
+            string path = Path.Combine(unitTestFolder, "Data", "SingleRight", "DelegationAccessCheckResponse");
+            string content = File.ReadAllText(Path.Combine(path, $"{fileName}.json"));
```

Comment:
```suggestion
            if (File.Exists(path))
            {
            string content = File.ReadAllText(Path.Combine(path, $"{fileName}.json"));
```

This should probably be wrapped in a check if the file exists. All the way down to (and containing) the deserialization of the content